assets/src/components/mapPage.tsx

```diff
@@ -323,6 +324,10 @@ const MapPage = (): ReactElement<HTMLDivElement> => {
       useContext(StateDispatchContext),
     { selectedVehicleId = null } = searchPageState
 
+  useEffect(() => {
+    dispatch(closeView())
+  }, [dispatch])
+
```

Comment:
question: I may have outdated memory on this, but my expectation was that if you open the VPP on the ladder page, go to map page, then back to the ladder page, the VPP would still be open. Is that no longer the expected behavior?

Reply:
Ah good catch, I forgot that `closeView` also closes the VPP. I'll have to revisit that logic.

Reply:
Okay should be fixed in 2774ae4c.

Reply:
Appreciate the comment added too!